stretch_core/launch/stretch_scan_matcher.launch

```diff
@@ -2,11 +2,12 @@
 
   <!-- laser_scan_matcher -->
   <node pkg="laser_scan_matcher" type="laser_scan_matcher_node" name="laser_scan_matcher_node" output="screen">
-    <remap from="odom" to="odometry/filtered"/>
+    <!-- <remap from="odom" to="odometry/filtered"/> -->
+    <remap from="scan" to="scan_filtered"/>
     
     <param name="max_iterations" value="10"/>
-    <!--<param name="fixed_frame" value="map"/>-->
-    <param name="fixed_frame" value="odom"/>
+    <param name="fixed_frame" value="map"/>
```

Comment:
@hello-vinitha I would expect Lidar odometry to publish w.r.t. the odom frame since it experiences drift, right? Here's the REP on odom/map frames: https://www.ros.org/reps/rep-0105.html#odom

Reply:
@hello-binit yes, you're correct. I pushed a commit where I reverted the change.